.travis.yml

```diff
@@ -83,15 +96,19 @@ script:
   - moodle-plugin-ci phplint
   - moodle-plugin-ci phpcpd
   - moodle-plugin-ci phpmd
-  #- moodle-plugin-ci codechecker
+  - moodle-plugin-ci codechecker
   - moodle-plugin-ci validate
   - moodle-plugin-ci phpunit
   # Specific for version 1
-  - if [ ${MOODLE_PLUGIN_CI} = "1" ]; then moodle-plugin-ci csslint; fi
-  - if [ ${MOODLE_PLUGIN_CI} = "1" ]; then moodle-plugin-ci shifter; fi
-  - if [ ${MOODLE_PLUGIN_CI} = "1" ]; then moodle-plugin-ci jshint; fi
+  - 'if [ ${MOODLE_PLUGIN_CI} = "1" ]; then
```

Comment:
I also changed this because we have more visibility of the error than if everything is in one block.

Reply:
^^ all of this was because of the conflict.. I wanted to keep my local file and I didn't notice the changes I'm sorry